cosmic/src/comenv.f

```diff
@@ -87,24 +86,24 @@ SUBROUTINE COMENV(M01,M1,MC1,AJ1,JSPIN1,KW1,
       CALL star(KW1,M01,M1,TM1,TN,TSCLS1,LUMS,GB,ZPARS)
       CALL hrdiag(M01,AJ1,M1,TM1,TN,TSCLS1,LUMS,GB,ZPARS,
      &            R1,L1,KW1,MC1,RC1,MENV,RENV,K21,ST_tide,
-     &            ecsn,ecsn_mlow,1)
+     &            1)
       OSPIN1 = JSPIN1/(K21*R1*R1*(M1-MC1)+K3*RC1*RC1*MC1)
       MENVD = MENV/(M1-MC1)
       RZAMS = RZAMSF(M01)
 *
 * Decide which CE prescription to use based on LAMBDA flag
 * MJZ: NOTE - Nanjing lambda prescription DOES NOT WORK!
 *
-      IF(LAMBDA.EQ.1.0)THEN
-         LAMB1 = CELAMF(KW,M01,L1,R1,RZAMS,MENVD,LAMBDA)
-      ELSEIF(LAMBDA.LT.0.d0)THEN
-         LAMB1 = -1.0*LAMBDA
+      IF(LAMBDAF.EQ.1.0)THEN
```

Comment:
@carlrodriguez should make a not that we have changed LAMBDA to LAMBDAF

Reply:
Good to know, thanks.  I'm going to go through the const_bse.h file and every common variable to make sure there's still 1-to-1 correspondance.